README.md

```diff
@@ -19,7 +18,49 @@ In summary:
   This is a single extract for variables that are not expected to change, such as ethnicity.
 * Use cohort-joiner to join the cohorts.
```

Comment:
May be useful to have a bullet here describing what the expected output of running the action is

Reply:
Thanks, @LFISHER7. 074cd22 describes the expected outputs.